### DISCUSSION TITLE
What purpose does the random char on the GitHub query string serve?

### DISCUSSION THREAD

**Discussion by savagedev** [ezeoleaf/larry#64] [Q&A] [answered]
I see a randomChar being added to the GitHub query string, what purpose does this serve?

**Comment by ezeoleaf** [comment, marked as answer]
So, when using the search API from Github, if you search by a topic or language, you always get the same 1000 repositories (the best ranked for example).
If we add a randomChar to our call to the searchAPI we add even more randomness. Otherwise we were always picking a repository from the same 1000. Now, if randomChar is send to the searchAPI, we get a whole different set of 1000 repositories.
It was added to improve randomness and the publishing of repositories with not so many stars.

Does this make sense?
